src/app/repository/repository.service.ts

```diff
@@ -80,8 +84,17 @@ export class DataRepository {
     increment: number,
   ): GameChartData {
     const incrementedTime = lastRequestTime + increment * 1000;
-    const randomValue = Math.round(Math.random() * 10);
-
+    const randomValue = this.shouldIncreaseOrDecrease(this.lastChartValue);
+    this.lastChartValue = randomValue;
     return { t: new Date(incrementedTime), y: randomValue };
   }
+
+  private shouldIncreaseOrDecrease(previousValue: number): number {
+    const randomChance = Math.round(Math.random());
+    if (randomChance === 1 && previousValue !== 1) {
```

Comment:
```suggestion
    return previousValue + vector * (Math.random() / 4);
```
So step will be not fixed 0.1, but random between 0.0 and 0.25